docs/helm/README.md

```diff
@@ -93,12 +120,89 @@
 * **Success Response:**
   
   * **Code:** 200 <br />
+  * JSON encoded [Release structure](https://github.com/helm/helm/blob/master/pkg/release/release.go#L22)
+ 
+* **Error Response:**
+
+  * **Code:** 403 BAD REQUEST <br />
+    **Content:** `{ error : "error message" }`
+
+
+**Upgrade Helm Release**
+----
+  _Upgrade Helm release_
+
+* **URL**
+
+    `/api/helm/release`
+
+* **Method:**
+  
+  `PUT` 
+  
+*  **Put Data Params**
+
+```
+ {
+   name: [string],
+   namespace: [string]
+   chart_url: [string]
+   values: map[string]interface{}
+  }
+```
+
+*  **Example Request**
+```
+    {
+    	"name": "test-helm-release",
+    	"namespace": "default",
+    	"chart_url": "https://github.com/akashshinde/console/raw/helm_endpoints/pkg/helm/testdata/influxdb-3.0.2.tgz"
+        "values": { "service": {"type": "ClusterIP"} }
+    }
+```
+
+* **Success Response:**
+  
+  * **Code:** 200 <br />
+  * JSON encoded [Release structure](https://github.com/helm/helm/blob/master/pkg/release/release.go#L22)
  
 * **Error Response:**
 
   * **Code:** 403 BAD REQUEST <br />
     **Content:** `{ error : "error message" }`
 
+
+**Rollback Helm Release**
+----
+  _Rollback Helm release_
+
+* **URL**
+
+    `/api/helm/release`
+
+* **Method:**
+  
+  `PATCH` 
+  
+*  **Put Data Params**
+
+```
+ {
+   name: [string],
+   namespace: [string]
+   version: [int]
+  }
+```
+
+*  **Example Request**
+```
+    {
+    	"name": "test-helm-release",
+    	"namespace": "default",
+        "version": 1,
+    }
+```
+
```

Comment:
how does response look like? please document.

also, no doc about the release uninstall.

Reply:
Done